src/components/Firestore/Collection.tsx

```diff
@@ -146,7 +146,8 @@ export const Collection: React.FC<Props> = ({ collection }) => {
       <Route
         path={`${url}/:id`}
         render={({ match }: any) => {
-          const docRef = collection.doc(match.params.id);
+          const docId = decodeURIComponent(match.params.id);
```

Comment:
I think react router should already decode this for us, no? Please double check to make sure we're not double decoding.

Reply:
@edupsousa I added a document with percent-encoded '/'(slash) included at Id. (e.g. 'hello%2fworld')
And it finally works without `decodeURIComponent()` in this line.
(@minhoryang)